src/tools/pub/dependencies.md

```diff
@@ -396,6 +396,9 @@ dependency_overrides:
   may break your application.
 {{site.alert.end}}
 
+Only dependency overrides from he root pubspec are taken into account
+when doing package resolution. 
```

Comment:
I think what we want to clearly state is that `dependency_overrides` from the root package is considered.
But `dependency_overrides` inside a package that you depend on will be ignored.

Which means when you publish a package to pub.dev `dependency_overrides` will be ignored, by anyone who uses the package.

Reply:
@sigurdm Any thoughts on the above comments?

I think including this mention is a great idea, Jonas's first sentence particularly makes sense.

Reply:
Yeah - I took it as implied - but it is totally worth highlighting!